test/unit/discovery_attribute_set_test.rb

```diff
@@ -1,4 +1,5 @@
-require 'test_helper'
```

Comment:
And core `test_helper` required plugins `test_helper`. You need to be this explicit.